frontend/src/pages/projects/notebook/NotebookActionsColumn.tsx

```diff
@@ -82,55 +74,70 @@ const NotebookStatusToggle: React.FC<NotebookStatusToggleProps> = ({
     });
   }, [fireNotebookTrackingEvent, notebookName, notebookNamespace, refresh]);
 
-  return (
-    <Flex gap={{ default: 'gapSm' }} alignItems={{ default: 'alignItemsCenter' }}>
-      <Switch
-        aria-label={label}
-        data-testid="notebook-status-switch"
-        isDisabled={inProgress || isStopping || isDisabled}
-        id={`${notebookName}-${notebookNamespace}`}
-        isChecked={isRunningOrStarting}
-        onClick={() => {
-          if (isRunningOrStarting) {
-            if (dontShowModalValue) {
+  return [
+    <>
+      <ActionsColumn
+        items={[
+          {
+            isDisabled: isDisabled || isRunningOrStarting,
+            title: 'Start',
+            onClick: () => {
+              setInProgress(true);
+              const tolerationSettings = computeNotebooksTolerations(
+                dashboardConfig,
+                notebookState.notebook,
+              );
+              startNotebook(
+                notebook,
+                tolerationSettings,
+                enablePipelines && !currentlyHasPipelines(notebook),
+              ).then(() => {
+                fireNotebookTrackingEvent('started');
+                refresh().then(() => setInProgress(false));
+              });
+            },
+          },
+          {
+            isDisabled: isDisabled || !isRunningOrStarting,
+            title: 'Stop',
+            onClick: () => {
+              if (dontShowModalValue) {
+                handleStop();
+              } else {
+                setOpenConfirm(true);
+              }
+            },
+          },
+          {
+            isDisabled: isStarting || isStopping,
+            title: 'Edit workbench',
+            onClick: () => {
+              navigate(
+                `/projects/${project.metadata.name}/spawner/${notebookState.notebook.metadata.name}`,
+              );
+            },
+          },
```

Comment:
If I cancel out of editing the workbench, I don't end up back at the project list view.
If I press the browser back button, I end up on the project list view however I am on the wrong page or the expanded section is no longer open.

Reply:
Updated the spawner paged to navigate back on a cancel